elements/pl-dropdown/info.json

```
@@ -0,0 +1,8 @@
+{
+    "controller": "pl-dropdown.py",
+    "dependencies": {
+        "elementStyles": [
+            "pl-dropdown.css"
+        ]
+    }
```

```suggestion
    "controller": "pl-dropdown.py"
```
Looks like this got accidentally readded in f12eaeb3?

---

We need it to do some additional CSS overlay work.